pkg/golinters/gci.go

```diff
@@ -32,11 +33,32 @@ func NewGci() *goanalysis.Linter {
 		nil,
 	).WithContextSetter(func(lintCtx *linter.Context) {
 		localFlag := lintCtx.Settings().Gci.LocalPrefixes
+
+		if localFlag == "" && lintCtx.Settings().Gci.LocalPrefixModule {
+			var modules []string
+
+			for _, module := range lintCtx.Modules {
+				modules = append(modules, module.Path)
+			}
+
+			localFlag = strings.Join(modules, ",")
+		}
```

Comment:
this block looks redundant, the fallback to goimports and next same block will never be executed